src/test/components/TrackContextMenu.test.js

```diff
@@ -86,6 +86,8 @@ describe('timeline/TrackContextMenu', function() {
         getByText(/Hide other screenshot tracks/);
       const hideContentProcess = () => getByText(/Hide "Content Process"/);
 
+      const hideAllTracksByType = () => getByText(/Hide "Content Process"/);
```

Comment:
This still only hides one track. Once you create your new function, the text should be something like this: `Hide all "Content Process" tracks`.

Reply:
Hope I had fixed all issues as you mentioned.